cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -754,38 +754,148 @@
 
   <property>
     <name>kafka.default.replication.factor</name>
+    <value>${kafka.server.default.replication.factor}</value>
+    <description>
+      CDAP Kafka service replication factor (deprecated: replaced with kafka.server.replication.factor)
+    </description>
+  </property>
+
+  <property>
+    <name>kafka.log.dirs</name>
+    <value>${kafka.server.log.dirs}</value>
+    <description>
+      CDAP Kafka service log storage directory (deprecated: replaced with kafka.server.log.dir)
+    </description>
+  </property>
+
+  <property>
+    <name>kafka.log.flush.interval.messages</name>
```

Comment:
for the `kafka.server.*` keys, if we want a different default than the one comes with Kafka, then we can keep them in cdap-default.xml